data/articles/201609/8.json

```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+        "name": "Percy Mamedy",
+        "twitter": "PercyMamedy"
+    },
+    "website": {
+        "name": "DotDev.co",
+        "site_url": "dotdev.co",
+        "feed_url": ""
+    },
+    "category": {
+        "id": 9,
+        "name": "Laravel",
+        "description": "Articles related to Laravel framework"
+    },
+    "url": "https:\/\/dotdev.co\/understanding-laravel-service-container-bd488ca05280#.d6xvxxaxb",
+    "title": "Understanding the Laravel Service Container",
+    "published_at": "2016-09-12",
+    "description": "On devdot.co blog, Percy wrote a small but nice post describing Laravel service container. In Laravel, services is the right place to write business logic. In his post, Percy is explaining how to use Service container (or IoC Container) to decouple your code and manage dependencies.",
+    "json_file_name": "201609\/8.json"
+}
```

Comment:
Please do not add articles with this issue. Remove file from here.

We add articles to article branch. I'll tell you how to add new articles.

Reason: It will impact article release process.